rfc/0001-rfc-process.md

```diff
@@ -23,68 +23,96 @@ The RFC process is designed to be lightweight, collaborative and effective. Doin
+Two examples: if the java client includes a helper class for retry semantics on RxJava observables (to reduce the verbosity in writing it manually), this does not have an equivalent on any other SDK and therefore doesn't need a RFC. If the java client introduces a helper class for LDAP support, this does have an impact on the other SDKs and needs a RFC.
 
-When it is not clear if an RFC needs to be created, a quick email or discussion in the team should make it clear wheter the change has an overall impact or not.
+When it is not clear if an RFC needs to be created, a quick email or discussion in the team should make it clear whether the change has an overall impact or not.
 
 **An RFC MUST be signed off by each SDK representative and consensus must be reached. Once the RFC is accepted, each SDK MUST implement it according to the RFC. If, after acceptance issues show up during implementation, the MUST lead to a revision of the RFC which again needs to be signed off by everyone.**
 
 # General Design
 
- 1. Look at latest issue to determine the rfc number (4 digits) - the `RFCID` is in the form *`rfcNumber`***-***`rfcShortName`*
- 2. open an issue titled `RFCID`, short description of the rfc, does the owner think it requires sdk specifics, etc... tag as appropriate (if we decide to put tags in). the issue allows to "reserve" the rfc number prior to any file being added publicly to the repo.
- 3. create the rfc branch: `drafts/RFCID`
- 4. create the rfc file as `RFCID.md`in `/rfc` directory
- 5. start the discussion by doing a Pull Request to master
- 6. **phase one** iterates on the global idea + "proof of concept" sdk specifics proposed by the owner
- 7. *end of this phase should probably be marked by a first informal round of votes?*
- 8. **phase two** iterates on sdk specifics: each sdk team/representative can then push commits in the branch to improve on his own sdk specifics
- 9. once everyone has contributed sdk specifics, the RFC must be signed off by a message following this convention:
- 
+The RFC must always be in one of the following states, and transition through them in the given order:
+
+ 1. *Identified*: The need for an RFC has been identified, but none actually drafted yet.
+ 2. *Draft*: The RFC is currently being drafted and subject to heavy modifications.
+ 3. *Review*: The RFC is believed to be complete (by stakeholders) and is broadcasted for a wider circle for final remarks. Relevant SDKs need to fully implement the RFC before it can be moved into Accepted.
+ 4. *Accepted*: The RFC is accepted, no further changes can be made to this RFC version. Subsequent changes need to be lifecycled again.
```

Comment:
@daschl could you specify explicitly whether or not new RFCID have to be used when subsequence changes need to be applied to *Accepted* RFC?